week-2-practise/level-1/2-validation.js

```diff
@@ -2,7 +2,7 @@
 // TIP: You should write an expression that returns a boolean value
 
 var num = 10;
-var isBigEnough; // ONLY EDIT THIS LINE
+var isBigEnough = num; // ONLY EDIT THIS LINE
```

Comment:
I don't think this is quite correct. The exercise is asking you to calculate if the `num` variable is bigger or equal than 10. If you change the `num` variable to 5 it will still log "num is bigger than or equal to 10". How could you fix that?